Neos.Media/Migrations/Mysql/Version20200823164700.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Migrations\AbortMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
```

Comment:
```suggestion
use Doctrine\DBAL\Migrations\AbstractMigration;
```
Changed to the abstractMigration of the used Doctrine Version